libs/providers/config-cat/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@openfeature/config-cat-provider",
+  "version": "0.0.1",
```

Comment:
This version doesn't match the version in the [release-please manifest](https://github.com/open-feature/js-sdk-contrib/pull/334/files#diff-e38bb01e41ed5baa3c9e8111b86009ca62d4498d2aefd14d367cd6300dd8b126). That's not really a huge deal, the first release will "correct" it but I'll point it out just in case you care about the version number.

Reply:
Changed it.
@toddbaert which version would you recommend to start with? I was not really sure when looking at the other packages.